auslib/test/admin/views/test_releases.py

```diff
@@ -1281,6 +1330,22 @@ def testEnactScheduledChangeNewRelease(self):
         }
         self.assertEquals(base_row, base_expected)
 
+    def testEnactScheduledChangeDeleteRelease(self):
+        ret = self._post("/scheduled_changes/releases/4/enact")
+        self.assertEquals(ret.status_code, 200, ret.data)
+
+        r = dbo.releases.scheduled_changes.t.select().where(dbo.releases.scheduled_changes.sc_id == 4).execute().fetchall()
+        self.assertEquals(len(r), 1)
+        db_data = dict(r[0])
+        expected = {
+            "sc_id": 4, "complete": True, "data_version": 2, "scheduled_by": "bill", "change_type": "delete", "base_name": "ab", "base_product": None,
+            "base_read_only": False, "base_data": None, "base_data_version": 1,
+        }
+        self.assertEquals(db_data, expected)
+
+        base_row = dbo.releases.t.select().where(dbo.releases.name == "a").execute().fetchall()
+        self.assertEquals(len(base_row), 1)
```

Comment:
Was expecting a check for the "ab" not existing here first, then you're checking that "a" didn't get taken out as collateral damage ?